tools/msvs/msi/nodemsi.wixproj

```diff
@@ -54,9 +54,12 @@
     </WixExtension>
   </ItemGroup>
   <ItemGroup>
-    <!-- <EmbeddedResource Include="i18n\de-de.wxl" /> -->
-    <EmbeddedResource Include="i18n\en-us.wxl" />
+    <!--
```

Comment:
@joaocgreis @fhemberger   is using weekly build of WiX a viable option for building node installer?